Inspicio/Controllers/AccountController.cs

```diff
@@ -122,6 +122,12 @@ public IActionResult Register(string returnUrl = null)
             if (ModelState.IsValid)
             {
                 var user = new ApplicationUser { UserName = model.Email, Email = model.Email };
```

Comment:
Also, once you've got a style you like it'll be a good idea to include an .editorconfig in the solution folder so the tools will encourage you to follow it.  If you do decided to go with the usual C# style, then you could just copy the one from Rosyln https://github.com/dotnet/roslyn/blob/master/.editorconfig